smartsim/settings/arguments/launch/dragon.py

```diff
@@ -71,7 +71,7 @@ def set(self, key: str, value: str | None) -> None:
         """
         set_check_input(key, value)
         if key in self._launch_args and key != self._launch_args[key]:
```

Comment:
Also somehow got past a previous review
```suggestion
        if key in self._launch_args and value != self._launch_args[key]:
```
?